tests/unit/s2n_kem_preferences_test.c

```diff
@@ -34,6 +36,8 @@ int main(int argc, char **argv)
 
     {
         const struct s2n_kem_group *test_kem_groups[] = {
+            &s2n_secp256r1_mlkem_768,
+            &s2n_x25519_mlkem_768,
```

Comment:
Any reason this needs to be another separate list and can't be ALL_SUPPORTED_KEM_GROUPS?

Reply:
Fixed

Reply:
Was this missed?

Reply:
Yes, you're right. I made this fix to use `ALL_SUPPORTED_KEM_GROUPS` in `s2n_kem_preferences.c`, but not `s2n_kem_preferences_test.c`.